photon-server/build.gradle

```diff
@@ -130,6 +130,7 @@ task generateJavaDocs(type: Javadoc) {
 
 spotless {
     java {
+        lineEndings='unix' 
```

Comment:
Do we want to force unix line endings?

Reply:
@Bankst

Reply:
This was the weird magic sauce which allowed `spotlessApply` to do the right thing on my windows machine. [Seems to be "not recommended' for some reason](http://mail-archives.apache.org/mod_mbox/geode-dev/201811.mbox/%3CCAFXXiPDRuDnwCr_rG+n8CNZPp4TT1+2-kjAtuTV+U5+FbVqdhg@mail.gmail.com%3E), but no idea if this is the real answer. We could pull it back out if needed - that should make spotless do the right thing in CI (just not on my machine).

Still befuddled by the actual root cause of the issues with spotless. I'm wondering if it's because I usually configure git to do checkout/checkin as-is ? Seems weird that spotless/gradle would directly reference something in a git config, but not entirely improbable....

Reply:
[I stand corrected, referencing git files is exactly what spotless does](https://diffplug.github.io/spotless/javadoc/2.3.0/com/diffplug/gradle/spotless/LineEnding.html#GIT_ATTRIBUTES).

One more try, will revert this back and try to fix on my end using git configurations.

Reply:
FWIW: `git config --local core.eol LF` should be a different magic secret sauce one can run in their repo to get spotless doing the right thing in the particular local repo.